cmd/neblictl/internal/controlplane/commands.go

```diff
@@ -325,6 +325,27 @@ func NewCommands(controlPlaneExecutors *Executors, controlPlaneCompleters *Compl
 						Optional: true,
 						Default:  "false",
 					},
+					{
+						Name:        "keyed",
+						Description: "Convert stream to a keyed stream. During the computation of an event in a keyed stream, each different key will have its own state",
```

Comment:
I would say `Define as a keyed stream` or `Configure` or similar instead of `Convert`. It has still not been created so it is not really converting anything I'd say. For consistency and to make thing easy I would change the option in the update message as well.